core/src/main/scala/filodb.core/memstore/OnDemandPagingShard.scala

```diff
@@ -33,6 +34,9 @@ TimeSeriesShard(dataset, storeConfig, shardNum, rawStore, metastore, evictionPol
                 downsampleConfig, downsamplePublisher)(ec) {
   import TimeSeriesShard._
 
+  private val tags = Map("shard" -> shardNum.toString, "dataset" -> dataset.toString)
+  private val inMemoryPartitionsMetric = Kamon.counter("odp-shard-inmemory-partitions").refine(tags)
+  private val opdPartitionsMetric = Kamon.counter("odp-shard-odp-partitions").refine(tags)
```

Comment:
I see possible duplicates. We already have partitionsPagedFromColStore in TimeSeriesShardStats. Please check if any can be reused.